src/mediapipe_calculators/python_executor_calculator.cc

```diff
@@ -37,37 +37,63 @@ namespace mediapipe {
 const std::string INPUT_SIDE_PACKET_TAG = "PYTHON_NODE_RESOURCES";
 
 class PythonExecutorCalculator : public CalculatorBase {
-    /* 
-    TODO: Streaming support:
-        - timestamping
-        - loopback input
-    */
     std::shared_ptr<PythonNodeResource> nodeResources;
     std::unique_ptr<PyObjectWrapper<py::iterator>> pyIteratorPtr;
+    // The calculator manages timestamp for outputs to work independently of inputs
+    // this way we can support timestamp continuity for more than one request in streaming scenario.
+    mediapipe::Timestamp outputsTimestamp;
```

Comment:
```suggestion
    mediapipe::Timestamp nextOutputTimestamp;
```
Would it be more descriptive?